controllers/config/defaults.go

```diff
@@ -3,6 +3,7 @@ package config
 const defaultTopic = "platform.inventory.events"
 const defaultConnectCluster = "xjoin-kafka-connect-strimzi"
```

Comment:
Should move to using `insights-kafka-connect` because that's what Devprod has moved to (per @SteveHNH ) because `xjoin` is not used anymore.  This frees us up from the maintenance of `xjoin-kafka-connect-strimzi` image.  May be this should be done in a different PR

Reply:
Yes, I think that's a change for a different PR. It would probably make more sense to be done as part of the xjoin-kafka-connect-strimzi vulnerabilities Jira